src/OpenPlatform.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace DDB\OpenPlatform;
+
+use DDB\OpenPlatform\Request\GenericRequest;
+use DDB\OpenPlatform\Request\SearchRequest;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class OpenPlatform
```

Comment:
I would consider calling this `Client` - and perhaps the library in itself `openplatform-client` since the project in itself does in fact represent OpenPlatform. Naming is hard...

Reply:
I've always hated `Guzzle\Client`... You end up with methods that require a `Client` unless you rename the import.

I feel the same way about Client as I do about Interface, Trait and Exception. I know that `OpenPlatform` is a client, but I really don't care as a user.

The package name on the other hand makes sense to me. `composer install d25k/openplatorm` looks like installing the service.